scss/_patterns_code-snippet.scss

```diff
@@ -0,0 +1,25 @@
+@import 'settings';
+
+@mixin vf-p-code-snippet {
+  .p-code-snippet {
+    .p-code-snippet__block {
+      border: 0;
+      border-radius: 0;
+      margin: 0;
+    }
+
+    .p-code-snippet__header {
+      background-color: rgba(0, 0, 0, 0.08);
```

Comment:
@lyubomir-popov Is the code snippet header background one of our existing colors? Or should we add new one?

This should use a color variable, not inline value.